example.fi

```diff
@@ -10,6 +10,12 @@ fn hello : String do
   "World"
 end
 
+# Anonymous function
+fn anonymous_call : Int do
+  f = (x: Int, y: Int){ x + y }
```

Comment:
@emilsoman wouldn't it be better to have anonymous functions have the same block delimiters as names functions?

Reply:
Absolutely, we'll keep it consistent. I'm going to change named functions to use `{}` as well. I thought there was no need to keep do-end here only to change it later 😄

Reply:
On second thought, it's good to keep it consistent for now, so i've changed this to do-end. I need some more feedback from potential first users before making the switch to `{}`.

Reply:
LGTM